pkg/gui/mergeconflicts/rendering.go

```diff
@@ -35,6 +35,19 @@ func shiftConflict(conflicts []*mergeConflict) (*mergeConflict, []*mergeConflict
 	return conflicts[0], conflicts[1:]
 }
 
-func shouldHighlightLine(index int, conflict *mergeConflict, top bool) bool {
-	return (index >= conflict.start && index <= conflict.middle && top) || (index >= conflict.middle && index <= conflict.end && !top)
+func shouldHighlightLine(index int, conflict *mergeConflict, selection Selection) bool {
+	switch selection {
+	case TOP:
+		if conflict.ancestor >= 0 {
```

Comment:
could we extract `conflict.ancestor >= 0` into a method called `hasAncestor` on the mergeConflict struct, to make the meaning a little clearer?